src/util/fork_utils.js

```diff
@@ -29,6 +31,15 @@ function start_workers(count = 0) {
         cluster.on('exit', (worker, code, signal) => {
             console.warn('WORKER exit', { id: worker.id, pid: worker.process.pid, code, signal });
             console.error('EXIT ON WORKER ERROR');
+            dbg.event({
```

Comment:
can you please add this reason to the first section of non_containzerized_NSFS_events.md please?
also please add to non_containzerized_NSFS.md the manual handling of this issue

Reply:
not relevant anymore - won't create an event - just restart a new one

Reply:
wait why deleting the event? we still want to notify regarding that fork restart
BTW, what about killing the main thread after a few retries?

Reply:
so as we discussed on slack, let's have a discussion on the restarts and add it in a different PR